runtimes/peregrine/src/kilt/mod.rs

```diff
@@ -21,9 +21,7 @@ use frame_system::{pallet_prelude::BlockNumberFor, EnsureRoot, EnsureSigned};
 use pallet_asset_switch::xcm::{AccountId32ToAccountId32JunctionConverter, MatchesSwitchPairXcmFeeFungibleAsset};
 use runtime_common::{
 	asset_switch::{hooks::RestrictSwitchDestinationToSelf, EnsureRootAsTreasury},
-	bonded_currencies::{
-		hooks::NextAssetIdGenerator, AssetId, FixedPoint, FixedPointInput, NativeAndForeignAssets, TargetFromLeft,
-	},
+	bonded_currencies::{self, hooks::NextAssetIdGenerator, AssetId, FixedPoint, FixedPointInput, TargetFromLeft},
```

Comment:
This is ugly 😄

Reply:
[baad1e3](https://github.com/KILTprotocol/kilt-node/pull/828/commits/baad1e3b5d31fbc465bca7f4e74ee6985a07a7a3)